inc/parserutils.php

```diff
@@ -115,6 +115,8 @@ function p_wiki_xhtml($id, $rev='', $excuse=true,$date_at=''){
 function p_locale_xhtml($id){
     //fetch parsed locale
     $html = p_cached_output(localeFN($id));
+    $data = ['id' => $id, 'html' => $html];
+    Event::createAndTrigger('PARSER_LOCALE_XHTML', $data);
     return $html;
```

Comment:
You return the original $html instead of the HTML modified by the plugin.

Reply:
I also wonder if the event shouldn't wrap around `p_cached_output(localeFN())` so plugins can also change the `$id` in the `BEFORE` event?